src/components/CallView/CallView.vue

```diff
@@ -373,6 +378,9 @@ export default {
 			return this.sharedDatas[this.shownRemoteScreenPeerId]?.remoteVideoBlocker?.isVideoEnabled()
 		},
 
+		overlaySize() {
+			return window.innerWidth * 0.1
```

Comment:
Not sure if it's intended (size hold proportion between presenter video / screen share), but on resize it's changing way too much. Top screenshot has around 0.4 of the width of bottom one. So I'd try to set min-max size limitations to it (component has `minw` and `minh` props) 

Also close icon size is not affected, but it's position is broken on resize (add position: absolute?)

![image](https://github.com/nextcloud/spreed/assets/93392545/1f7ea3ab-77af-4553-8993-c46eed3d10b1)

![image](https://github.com/nextcloud/spreed/assets/93392545/97b4df23-0d79-4299-9bad-759c88660bad)

Reply:
> Not sure if it's intended (size hold proportion between presenter video / screen share)

window.innerWidth is the full width of the whole page not only the screenshare div.